### DISCUSSION TITLE
Ped is Enemy to Player

### DISCUSSION THREAD

**Discussion by Rapscalli0n** [kibook/spooner#4] [General]
Thank you guys for this awesome script. I have a question though, is the ped config flag "Ped is Enemy to Player" working as intended? Even when I turn it on, the ped won't attack any of the players. Any other way to get peds to attack players?

**Comment by kibook** [comment]
Hi, and thanks for your interest!

The ped config flags are a native part of the game, and the menu in spooner just lets you toggle them on/off. I'm not sure exactly what effect all of them have. There's a little more information about some flags on this list: https://github.com/femga/rdr3_discoveries/tree/master/AI/CPED_CONFIG_FLAGS

If you give a ped a weapon, they will usually fight back if the player attacks them. I'm not sure if there's any simple flag you can set in the spooner to make them immediately hostile. There's natives like TASK_COMBAT_PED which will work, but currently aren't implemented in the spooner (I hope to add a more robust Task and Task Sequence-related menu in the future).

**Reply by Rapscalli0n** [reply]
Looking forward to it! If you ever find time do you know any pointers in making only certain ped models in your list fall under TaskCombatPed? I'm trying to find a duct tape solution since I'm pretty new to lua. I tried TaskCombatPed(entity, PlayerPedId()) but that turned my character in to a combat npc lol
